packages/ui/src/components/Tabs/api.ts

```diff
@@ -0,0 +1,27 @@
+import { forceInject } from '@/util'
+import { InjectionKey } from 'vue'
+import { InjectData } from './types'
+
+export interface STabPanelApi {
+  readonly modelValue: number
+  'update:modelValue': (num: number) => void
+}
+
+export interface STab {
+  'border-radius'?: 'full' | 'none' | 'right' | 'left'
+  disabled: boolean
+}
+
+export const STAB_PANEL_API_KEY: InjectionKey<STabPanelApi> = Symbol('STabPanelAPI')
+
+export function useSTabPanelApi(): STabPanelApi {
+  return forceInject(STAB_PANEL_API_KEY)
+}
+
+export const STAB_API_KEY: InjectionKey<STab> = Symbol('STabAPI')
+
+export function useSTabApi(): STab {
+  return forceInject(STAB_API_KEY)
+}
+
+export const tabsState: InjectionKey<InjectData> = Symbol('tabsState')

```

Comment:
- `STabsPanelApi` & `STab` is not used within components - what are their purpose?
- `STAB_PANEL_API_KEY` vs `tabsState` - inconsistent injection keys naming (please use CONSTANT_CASE and without `S`- prefix - this prefix is only needed for component names)